content/cybersecurity/concepts/virus/virus.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: 'Virus'
+Description: 'A virus is a type of computer program that can infect a system with malicious code and replicate itself.'
+Subjects:
+  - 'Cybersecurity'
+  - 'Computer Science'
+Tags:
+  - 'Cybersecurity'
+  - 'Computer Science'
+CatalogContent:
+  - 'paths/cyber-resilience-risk-management'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+A **virus** is a type of computer program that can infect a system with malicious code and replicate itself. Computer viruses have two main functions:
+
+1. Destruction refers to the damage a computer virus can cause to a computer by deleting files or reformatting the hard drive.
+2. Propagation is when a virus self-replicates by placing a copy of itself into other programs across a network.
+
+## Virus In-Use
+
+A virus's payload delivers the destructive power and negatively impacts the confidentiality, integrity, or availability of systems or data. Common signs of computer viruses can include unwanted pop-up windows, programs that self-execute, accounts being logged out, and crashing of the device. The risk of a virus can be mitigated by installing anti-virus software, scanning on a regular basis, and backing up data.
+
+## Different Types of Viruses
+
+- Malware: Short for malicious software, malware is any type of software that could harm a computer system. It can make a computer perform actions without the owner’s consent or knowledge.
```

Comment:
A virus is a type of malware, not the other way around

Reply:
@SSwiniarski I think we should make a differentiation between viruses, worms, and malware (the umbrella term).

@jonchery In this section on different types of viruses, I'm wondering if we can cover the different types of viruses like File infectors, Boot sector viruses, Polymorphic viruses, and Stealth viruses.